onnxruntime/core/mlas/lib/scalar/SconvDepthwiseKernelScalar.cpp

```diff
@@ -47,6 +47,7 @@ Routine Description:
 --*/
 {
     const size_t W = Parameters->InputShape[1];
+    bool beta_not_zero = Parameters->Beta == 1.0f;
```

Comment:
what if Beta is 0.5?

Reply:
I am sorry, but why would we get Beta of 0.5?  There are only two cases for Beta: accumulated or not, is that right? please correct me.

Reply:
I see.  Generized `Beta` parament.